components/common/AppMenu.vue

```diff
@@ -22,15 +22,15 @@
         <i class="material-icons notranslate">chevron_right</i>
       </nuxt-link>
 
-      <!-- <nuxt-link
-        class="app-menu-item "
-        to="/proposals"
+      <nuxt-link
+        class="app-menu-item"
+        to="/governance"
```

Comment:
```suggestion
        to="/proposals"
```

Reply:
you need to rename the folder. I'm going to do it